native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -532,6 +532,11 @@ int AndroidPlatform::init() {
     cc::FileUtilsAndroid::setAssetManager(_app->activity->assetManager);
     _inputProxy = ccnew GameInputProxy(this);
     _inputProxy->registerAppEventCallback([this](int32_t cmd) {
+        IXRInterface *xr = getInterface<IXRInterface>();
```

Comment:
I think should unify the codes, use `getInterface<IXRInterface>()` or `CC_GET_XR_INTERFACE()`.

Reply:
Currently, there are both used in this file.

Reply:
And it is better to use the macro `CC_USE_XR ` to enable xr related codes.

Reply:
new commit has replaced getInterface<IXRInterface>() with CC_GET_XR_INTERFACE()